opentelemetry/CHANGELOG.md

```diff
@@ -10,6 +10,10 @@
 
 - Remove `urlencoding` crate dependency. [#1613](https://github.com/open-telemetry/opentelemetry-rust/pull/1613)
 
+### Changed
+
+- Deprecate `versioned_logger()` and `versioned_tracer()` in favor of `logger_builder()` and `tracer_builder()` respectively [1567](https://github.com/open-telemetry/opentelemetry-rust/pull/1567).
```

Comment:
could we make the changelog more self-contained? for example, we can include actual changes expected by endusers right in this changelog itself.
eg:
provider.versioned_logger(name,version,schema,attr) should now be replaced with provider.builder().name().version()....

^ something like this? In general, I am trying to see if we can make chanelogs more self-contained wherever feasible, so end users can just read changelog and understand the necessary changes. For bigger ones, of course, link to the PR number, and we can give details in the PR description.

Reply:
Something like ?
Before:
```rust
let tracer = provider.versioned_tracer(
    "my-tracer-name",
    Some("X.Y.Z"),
    Some("https://opentelemetry.io/schema/1.0.0""),
    Some(vec![KeyValue::new("key", "value")]),
);
```
After:
```rust     
let tracer = provider.tracer_builder("my-tracername")
    .with_version(env!("CARGO_PKG_VERSION"))
    .with_schema_url(semcov::SCHEMA_URL)
    .with_attributes(vec![KeyValue::new("key", "value")])
    .build();
```

Reply:
Updated the changelog -- please check it out